.circleci/config.yml

```diff
@@ -103,6 +104,17 @@ jobs:
               exit 1
             fi
 
+  integration-test-override-ci:
```

Comment:
What do you think about adding an additional package to the `package.json` and `package-lock.json`? I was thinking it might make it easier to check if the caching is working, as it's a little hard to tell from https://app.circleci.com/pipelines/github/CircleCI-Public/node-orb/506/workflows/52a174ce-63cc-40ea-b1a7-94841cae0e6d/jobs/3405/steps , but no worries if you've already verified it